### ISSUE TITLE
Money.split/2 cash option behaving wrong

### ISSUE THREAD

**Issue by maennchen** [kipcole9/money#56]
The [documentation](https://github.com/kipcole9/money/blob/master/lib/money.ex#L1061) mentions this example:

```elixir
iex> Money.round Money.new("123.7456", :CHF), cash: true
#Money<:CHF, 125>
```

The correct result should be `#Money<:CHF, 125.75>` though, since the smallest coin in CHF is `0.05` (5 centimes called "Rappen"). See in the table here: https://en.wikipedia.org/wiki/Swiss_franc#Coins

In accounting the smallest precision is `0.01` (1 centime).

**Comment by kipcole9** [comment]
Apologies, definitely a bug.  The formatting output is rounding correctly:
```
iex(12)> Money.to_string Money.new("123.7456", :CHF), cash: true
{:ok, "CHF123.75"}
```
But `Money.round/2` is not.  I will fix ASAP.

**Comment by kipcole9** [comment]
As [of this commit](https://github.com/kipcole9/money/commit/56a4b15ba49908240a4aad2dded1badd078d128c) the rounding is now correctly made. Note too that the option to specify cash digits rounding is `currency_digits: :cash`, not `cash: true`.  The updated release will be in hex in a few minutes.

## Changelog for Money v2.2.1

This is the changelog for Money v2.2.1 released on February 22, 2018.  For older changelogs please consult the release tag on [GitHub](https://github.com/kipcole9/money/tags)

### Bug Fixes

* Correctly round to cash increment Money.round/2 now correctly uses the rounding increment for a currency. This is relevant for currencies like :AUD and :CHF which have minimum cash rounding of 0.05 even though the accounting increment is 0.01.  Thanks to @maennchen.  Closes #56.

* Update documentation for `Money.round/2` to correctly refer to the option `:currency_digits` with the valid options of `:cash`, `:accounting` and `:iso`.  The default is `:iso`.  The option `cash: true` is invalid.
